ml-agents/mlagents/trainers/settings.py

```diff
@@ -600,9 +600,14 @@ def _team_change_default(self):
 class TrainerType(Enum):
     PPO: str = "ppo"
     SAC: str = "sac"
+    POCA: str = "poca"
 
     def to_settings(self) -> type:
-        _mapping = {TrainerType.PPO: PPOSettings, TrainerType.SAC: SACSettings}
+        _mapping = {
+            TrainerType.PPO: PPOSettings,
+            TrainerType.SAC: SACSettings,
+            TrainerType.POCA: PPOSettings,
```

Comment:
For clarity define POCA settings as 
```
POCASettings = PPOSettings
```
Will set us up better when the two settings diverge